src/mediaLibrary/MediaLibraryVideoClient.ts

```diff
@@ -0,0 +1,217 @@
+import {lastValueFrom, type Observable} from 'rxjs'
+
+import {_request} from '../data/dataMethods'
+import type {ObservableSanityClient, SanityClient} from '../SanityClient'
+import type {
+  HttpRequest,
+  MediaLibraryAssetInstanceIdentifier,
+  MediaLibraryPlaybackInfoOptions,
+  VideoPlaybackInfo,
+  VideoPlaybackInfoItem,
+  VideoPlaybackInfoItemSigned,
+  VideoPlaybackInfoSigned,
+  VideoPlaybackTokens,
+} from '../types'
+
+/** @internal */
+export class ObservableMediaLibraryVideoClient {
+  #client: ObservableSanityClient
+  #httpRequest: HttpRequest
+  constructor(client: ObservableSanityClient, httpRequest: HttpRequest) {
+    this.#client = client
+    this.#httpRequest = httpRequest
+  }
+
+  /**
+   * Get video playback information for a media library asset
+   *
+   * @param assetIdentifier - Asset instance identifier (GDR, video-prefixed ID, or container ID)
+   * @param options - Options for transformations and expiration
+   */
+  getPlaybackInfo(
+    assetIdentifier: MediaLibraryAssetInstanceIdentifier,
+    options: MediaLibraryPlaybackInfoOptions = {},
+  ): Observable<VideoPlaybackInfo> {
+    const {instanceId, libraryId} = parseAssetInstanceId(assetIdentifier)
+    const uri = buildVideoPlaybackInfoUrl(instanceId, libraryId)
+    const queryParams = buildQueryParams(options)
+
+    return _request<VideoPlaybackInfo>(this.#client, this.#httpRequest, {
+      method: 'GET',
+      uri,
+      query: queryParams,
+    })
+  }
+
+  /**
+   * Extract playback tokens from signed video playback info
+   *
+   * @param playbackInfo - The playback info response
+   * @returns Object containing playback, thumbnail, and storyboard tokens if signed, undefined otherwise
+   */
+  getPlaybackTokens(playbackInfo: VideoPlaybackInfo): VideoPlaybackTokens | undefined {
+    if (isSignedPlaybackInfo(playbackInfo)) {
+      return {
+        playback: playbackInfo.stream.token,
```

Comment:
What's the reasoning behind `stream` being aliased to `playback` here? I expected that the shape of this object would align with the properties in `VideoPlaybackInfo`.